docs/sql/indexes.md

```diff
@@ -10,13 +10,13 @@ DuckDB has two built-in index types. Indexes can also be defined via [extensions
 
 ### Min-Max Index (Zonemap)
 
-A [min-max index](https://en.wikipedia.org/wiki/Block_Range_Index) (also known as zonemap and block range index) is automatically created for columns of all [general-purpose data types]({% link docs/sql/data_types/overview.md %}).
+A [min-max index](https://en.wikipedia.org/wiki/Block_Range_Index) (also known as zonemap or block range index) is automatically created for columns of all [general-purpose data types]({% link docs/sql/data_types/overview.md %}).
 
 ### Adaptive Radix Tree (ART)
 
 An [Adaptive Radix Tree (ART)](https://db.in.tum.de/~leis/papers/ART.pdf) is mainly used to ensure primary key constraints and to speed up point and very highly selective (i.e., < 0.1%) queries. ART indexes are automatically created for columns with a `UNIQUE` or `PRIMARY KEY` constraint and can be defined using `CREATE INDEX`.
 
-> Warning ART indexes must currently be able to fit in-memory during index creation. Avoid creating ART indexes if the index does not fit in memory during index creation.
```

Comment:
removed unnecessary hyphen; also makes it consistent with "in memory" in the following sentence